lib/id3c/cli/command/etl/presence_absence.py

```diff
@@ -273,6 +273,9 @@ def update_sample(db: DatabaseSession,
 
     LOG.info(f"Found sample {sample.id} «{sample.identifier}»")
 
+    if sample.details.get("nwgc_id") and additional_details:
+        additional_details = update_details_nwgc_id(sample, additional_details)
+
```

Comment:
Might be good to be defensive against the case that `sample.details` is null/None.

Reply:
Not sure what you mean here. If `sample.details` is null/None, then it gets set to the value of `additional_details` which already contains `nwgc_id` as a list. I can add in a comment to make this more apparent.

Reply:
Ah, sorry, I should have used more words. I mean that if `sample.details` is `None` then `sample.details.get(…)` will fail.